inc/includes.php

```diff
@@ -181,4 +181,17 @@
 // Manage entity change
 if (isset($_REQUEST["force_entity"]) && ($_SESSION["glpiactive_entity"] ?? -1) != $_REQUEST["force_entity"]) {
     Session::changeActiveEntities($_REQUEST["force_entity"], true);
+} else {
+    $glpiactiveentities = $_SESSION['glpiactiveentities'] ?? [];
+    if (count($glpiactiveentities) > 1) {
```

Comment:
Relying on `glpiactive_entity_recursive` is safer. There is no button in the UI to use an entity tree on an entity that has no children, but maybe it is possible from the API or in a plugin.

```suggestion
    if (
        array_key_exists('glpiactive_entity_recursive', $_SESSION)
        && $_SESSION['glpiactive_entity_recursive']
    ) {
        $glpiactiveentities = $_SESSION['glpiactiveentities'];
```

Reply:
Indeed.